public/puzzles.js

```diff
@@ -2459,44 +2385,276 @@ const start = new Date(2022, 2 - 1, 10);
 
 const MILLIS_PER_DAY = 1000 * 60 * 60 * 24;
 
+/**
+ * @param {Date} first
+ * @param {Date} second
+ * @returns int
```

Comment:
these should be `number`, I guess - but let's let the script catch it 😄

Reply:
I think they are dates actually. I noticed they had problems when I loaded it into VS Code.

You can do `new Date() - new Date()` in javascript, but the type compiler doesn't like it.

Changed these to `new Date().getTime() - new Date().getTime()` which apparently is what the original gets coerced into.

Reply:
Oh, I just meant the type you're specifying, but I see now it's not a type (`{int}`) so I guess it's ok. Typescript only has `{number}`

Reply:
oops!